packages/webapp/pages/onboarding.tsx

```diff
@@ -142,7 +132,7 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.EditTag);
     }
 
-    if (activeScreen === OnboardingStep.EditTag && showReadingReminder) {
+    if (activeScreen === OnboardingStep.EditTag && isMobile) {
```

Comment:
Where was isMobile before or was that decision after experiment won?

Reply:
It was in growthbook side

<img width="757" alt="Screenshot 2024-07-18 at 11 31 42 AM" src="https://github.com/user-attachments/assets/3c54cd7c-3d04-413d-9511-035acbaae186">

Reply:
🚀